src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -889,6 +919,18 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         return _terminal->IsTrackingMouseInput();
     }
 
+    // Method Description:
+    // - Handles changes in mouse mode state
+    winrt::fire_and_forget TermControl::_TerminalMouseModeChanged()
+    {
+        co_await Dispatcher();

```

Comment:
Didn't think that would work without a param, that's neat.

Reply:
yeah! It's supes cool